deepmd/entrypoints/transfer_to_ascend.py

```diff
@@ -0,0 +1,13 @@
+from deepmd.utils.transfer_to_ascend import mix_precision
+
+def transfer_to_ascend(
+    *,
+    TO: str,
+    input_model: str,
+    output_model: str,
+    **kwargs,
+):
+    if TO == 'mix_precision':
+        mix_precision(input_model, output_model, **kwargs)
+    else:
+        raise RuntimeError('unsupported trasnfering version' + FROM)
```

Comment:
transfer

Reply:
It looks that `mix_precision` is a type instead a version.

Reply:
Thanks! we fixed these mistakes.